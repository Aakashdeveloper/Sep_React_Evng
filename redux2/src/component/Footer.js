import React from 'react';

const Footer = () => {
    return(
        <Footer>
            <div>
                <hr/>
                &copy; Developer Funnel
            </div>
        </Footer>
    )
}

export default Footer;