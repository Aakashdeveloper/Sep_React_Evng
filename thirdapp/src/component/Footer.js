import React from 'react';

const Footer = () => {
    return(
        <div>
            <center>
                <h4>&copy; Developer Funnel</h4>
            </center>

        </div>
    )
}

export default Footer;