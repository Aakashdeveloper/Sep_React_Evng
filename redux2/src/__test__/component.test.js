import React from 'react';
import Footer from '../component/Footer';
import {create} from 'react-test-renderer';
import BaseButton from '../component/BaseButton';


describe('SnapShot Footer Test',() => {
    test('Testing Footer',() => {
        let tree = create(<Footer/>)
        expect(tree.toJSON()).toMatchSnapshot()
    });

    test("it shows the expected text when clicked", () => {
      const component = create(<BaseButton/>);
      const instance = component.root;
      const button = instance.findByType("button");
      button.props.onClick();
      expect(button.props.children).toBe("Some Thing");
    });
})
