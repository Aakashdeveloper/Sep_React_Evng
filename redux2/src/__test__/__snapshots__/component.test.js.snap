// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapShot Footer Test Testing Footer 1`] = `
<footer>
  <div>
    <hr />
    Â© Developer Funnel
  </div>
</footer>
`;
